build/wd_cc_benchmark.bzl

```diff
@@ -23,6 +23,15 @@ def wd_cc_benchmark(
         ],
         # Only run benchmarks when explicitly requested, at least until we have some more of them
         # and can define a benchmark suite.
-        tags = ["manual", "benchmark"],
+        tags = ["off-by-default", "benchmark"],
         **kwargs
     )
+
+    # generate benchmark report
+    native.genrule(
+      name = name + "@benchmark.csv",
```

Comment:
You're probably aware, but the benchmark library also supports JSON output, just use `--benchmark_format=csv`. https://github.com/google/benchmark/blob/main/docs/tools.md describes a python script to compare test outputs which might be useful in the long term. This approach requires JSON inputs, so we can consider adding JSON if we plan to use that tool later on. For now just having CSV would be fine, it's arguably easier for humans to read.